api_tests/src/ledgers/geth_instance.ts

```diff
@@ -0,0 +1,220 @@
+import { ChildProcess, spawn } from "child_process";
+import waitForLogMessage from "../wait_for_log_message";
+import { existsAsync, mkdirAsync, openAsync, writeFileAsync } from "../utils";
+import getPort from "get-port";
+import { Logger } from "log4js";
+import { LedgerInstance } from "./index";
+import findCacheDir from "find-cache-dir";
+import download from "download";
+import { platform } from "os";
+import chmod from "chmod";
+import * as path from "path";
+
+export class GethInstance implements LedgerInstance {
+    private process: ChildProcess;
+    // The parameter --networkid does not seem to have an effect for dev chain.
+    // Nevertheless we define it so that it is obvious where the chain_id is coming from
+    public readonly CHAIN_ID = 1337;
```

Comment:
Note: As far as I understand the network ID would only be important if we would form a (private) network with another node. `dev` mode is single-node mode, so, as you stated, the `network-id` should not have an affect on it.

Reply:
Yes, I agree. I just wanted to change it to `17` because that's what we used before :) It's not that important for us though.